cpp/src/slice2js/Gen.h

```diff
@@ -21,7 +21,7 @@ namespace Slice
     protected:
         void writeMarshalDataMembers(const DataMemberList&, const DataMemberList&);
         void writeUnmarshalDataMembers(const DataMemberList&, const DataMemberList&);
-        void writeInitDataMembers(const DataMemberList&);
+        void writeOneShotConstructorArguments(const DataMemberList& members);
```

Comment:
`writeInitDataMembers` was 100% dead, not even implemented anywhere.
So. That's just free real estate there!